From 9eead096d5938ffe85e144084141d5ffdd5db660 Mon Sep 17 00:00:00 2001
From: "Ooi, Joyce" <joyce.ooi@intel.com>
Date: Thu, 30 Jun 2016 17:43:29 +0800
Subject: [PATCH 2487/2508] hid: intel-ish-hid: ipc: Update Intel SoC name

commit bff9a00566912a55518986dd72a26ee4a2953261 from
https://github.com/01org/linux-apollolake-i

Intel SoC name is updated to APL as APL is the official product code name.

Signed-off-by: Ooi, Joyce <joyce.ooi@intel.com>
---
 drivers/hid/intel-ish-hid/ipc/hw-ish.h  |    2 +-
 drivers/hid/intel-ish-hid/ipc/pci-ish.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/hid/intel-ish-hid/ipc/hw-ish.h b/drivers/hid/intel-ish-hid/ipc/hw-ish.h
index 7a35b41..46615a0 100644
--- a/drivers/hid/intel-ish-hid/ipc/hw-ish.h
+++ b/drivers/hid/intel-ish-hid/ipc/hw-ish.h
@@ -24,7 +24,7 @@
 #define CHV_DEVICE_ID		0x22D8
 #define BXT_Ax_DEVICE_ID	0x0AA2
 #define BXT_Bx_DEVICE_ID	0x1AA2
-#define BXTP_Ax_DEVICE_ID	0x5AA2
+#define APL_Ax_DEVICE_ID	0x5AA2
 #define SPT_Ax_DEVICE_ID	0x9D35
 
 #define	REVISION_ID_CHT_A0	0x6
diff --git a/drivers/hid/intel-ish-hid/ipc/pci-ish.c b/drivers/hid/intel-ish-hid/ipc/pci-ish.c
index 941a741..a57d187 100644
--- a/drivers/hid/intel-ish-hid/ipc/pci-ish.c
+++ b/drivers/hid/intel-ish-hid/ipc/pci-ish.c
@@ -34,7 +34,7 @@ static const struct pci_device_id ish_pci_tbl[] = {
 	{PCI_DEVICE(PCI_VENDOR_ID_INTEL, CHV_DEVICE_ID)},
 	{PCI_DEVICE(PCI_VENDOR_ID_INTEL, BXT_Ax_DEVICE_ID)},
 	{PCI_DEVICE(PCI_VENDOR_ID_INTEL, BXT_Bx_DEVICE_ID)},
-	{PCI_DEVICE(PCI_VENDOR_ID_INTEL, BXTP_Ax_DEVICE_ID)},
+	{PCI_DEVICE(PCI_VENDOR_ID_INTEL, APL_Ax_DEVICE_ID)},
 	{PCI_DEVICE(PCI_VENDOR_ID_INTEL, SPT_Ax_DEVICE_ID)},
 	{0, }
 };
-- 
1.7.5.4

